index.js

```diff
@@ -0,0 +1,31 @@
+let theBooks = [];
+
+const addButton = document.querySelector('.button');
+addButton.addEventListener('click', () => {
+  const bookContainer = document.createElement('div');
+  const titleBook = document.createElement('h5');
+  const authorBook = document.createElement('h5');
+  const removeButton = document.createElement('button');
+  const breakLine = document.createElement('hr');
+  const inputTitle = document.querySelector('.title');
+  const inputAuthor = document.querySelector('.author');
+  titleBook.innerHTML = inputTitle.value;
+  authorBook.innerHTML = inputAuthor.value;
+  removeButton.innerHTML = 'Remove';
+
+  bookContainer.append(titleBook, authorBook, removeButton, breakLine);
+  const bookList = document.querySelector('.book-list');
+  bookList.append(bookContainer);
+
+  theBooks.push({ title: inputTitle.value, author: inputAuthor.value });
+
+  window.localStorage.setItem('booksArray', JSON.stringify(theBooks));
+
+  function removeBook() {
+    theBooks = theBooks.filter((book) => book.title !== this.parentNode.childNodes[0].innerHTML);
+    localStorage.setItem('booksArray', JSON.stringify(theBooks));
+    this.parentNode.remove();
```

Comment:
- [ ] Due to the fact that you are comparing the book `title` and not a unique `id`, all books with the same title will be removed from the `localStorage` when one is removed. Please give a unique `id` value to every book created so that they can be easily removed.